www/redland-bindings/Portfile

```diff
@@ -1,8 +1,7 @@
 PortSystem 1.0
```

Comment:
please add a modeline as the first line:
```# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4```

and make sure the content of the Portfile conforms to it (i.e., replace all tabs in the variants with four spaces; align everything with four spaces).